src/Icons/CarrierUPS.tsx

```diff
@@ -8,17 +8,35 @@ interface Props {
-  </defs>
-</svg>
-
+const UPS: React.FunctionComponent<Props> = ({
+  width = '54',
+  height = '54',
+  fillColor = '#2D2926',
+  transform = '',
+  ...rest
+}) => (
+  <svg
+    width={width}
+    height={height}
+    viewBox="0 0 54 54"
+    transform={transform}
+    fill="none"
+    xmlns="http://www.w3.org/2000/svg"
+    xmlnsXlink="http://www.w3.org/1999/xlink"
+  >
+    <rect width={width} height={height} fill="url(#pattern0)" />
+    <defs>
+      <pattern id="pattern0" patternContentUnits="objectBoundingBox" width="1" height="1">
```

Comment:
can you make this id and then one a little below more unique? like `carrierUPS-pattern`

Reply:
but we also can't have two of them on the same page if we use `id`. So maybe there's a better solution with svgs where you don't need to reference the image